### DISCUSSION TITLE
我想重构Radio、Checkbox了，搞一搞支持自定义勾选图标的那种🙈🙈

### DISCUSSION THREAD

**Discussion by vtrbo** [ikun-svelte/ikun-ui#116] [Ideas]
好烦，没想好通讯用那种，context好呢，还是store好呢...

**Comment by baiwusanyu-c** [comment]
我偏向 context，store没必要，context更贴近且符合于在某个子组件树上的的数据流动，只要层级不太深且没有兄弟组件之间数据流动，就还好

**Reply by vtrbo** [reply]
其实我也更倾向于context，这样就可以搞radio-group了。
